sdk/cognitivelanguage/Azure.AI.Language.Conversations/README.md

```diff
@@ -1,56 +1,56 @@
+[Source code][conversationanalysis_client_src] | [Package (NuGet)][conversationanalysis_nuget_package] | [API reference documentation][conversationanalysis_refdocs] | [Product documentation][conversationanalysis_docs] | [Samples][conversationanalysis_samples]
 
 ## Getting started
 
 ### Install the package
 
-Install the Azure Cognitive Language Services Question Answering client library for .NET with [NuGet][nuget]:
+Install the Azure Cognitive Language Services Conversations client library for .NET with [NuGet][nuget]:
 
 ```powershell
-dotnet add package Azure.AI.Language.QuestionAnswering --prerelease
+dotnet add package Azure.AI.Language.Conversations --prerelease
 ```
 
 ### Prerequisites
 
 * An [Azure subscription][azure_subscription]
-* An existing Question Answering resource
+* An existing Text Analytics resource
 
 > Note: the new unified Cognitive Language Services are not currently available for deployment.
 
 ### Authenticate the client
 
-In order to interact with the Question Answering service, you'll need to create an instance of the [`QuestionAnsweringClient`][questionanswering_client_class] class. You will need an **endpoint**, and an **API key** to instantiate a client object. For more information regarding authenticating with Cognitive Services, see [Authenticate requests to Azure Cognitive Services][cognitive_auth].
+In order to interact with the Conversations service, you'll need to create an instance of the [`ConversationAnalysisClient`][conversationanalysis_client_class] class. You will need an **endpoint**, and an **API key** to instantiate a client object. For more information regarding authenticating with Cognitive Services, see [Authenticate requests to Azure Cognitive Services][cognitive_auth].
 
 #### Get an API key
 
-You can get the **endpoint** and an **API key** from the Cognitive Services resource or Question Answering resource in the [Azure Portal][azure_portal].
+You can get the **endpoint** and an **API key** from the Cognitive Services resource in the [Azure Portal][azure_portal].
 
-Alternatively, use the [Azure CLI][azure_cli] command shown below to get the API key from the Question Answering resource.
+Alternatively, use the [Azure CLI][azure_cli] command shown below to get the API key from the Cognitive Service resource.
 
 ```powershell
 az cognitiveservices account keys list --resource-group <resource-group-name> --name <resource-name>
 ```
 
-#### Create a QuestionAnsweringClient
+#### Create a ConversationAnalysisClient
 
-Once you've determined your **endpoint** and **API key** you can instantiate a `QuestionAnsweringClient`:
+Once you've determined your **endpoint** and **API key** you can instantiate a `ConversationAnalysisClient`:
 
-```C# Snippet:QuestionAnsweringClient_Create
+```C# Snippet:ConversationAnalysisClient_Create
```

Comment:
These code changes actually need to be made in the tests/Azure.AI.Language.Conversations.Tests/samples directory, then you run (rrom the repo root) `eng/scripts/CodeChecks.ps1 cognitivelanguage` to update them and make sure everything is correct. That's why this PR build is also failing.

Reply:
We do this to make sure 1) our samples actually compile, and 2) when recorded or run live, the samples actually work like we expect.

Reply:
...or are these from my previous changes?